src/context.rs

```diff
@@ -56,6 +56,8 @@ pub struct Game {
 enum ScheduledStop {
     /// Schedule a stop after `usize` step
     Step(usize),
```

Comment:
rename to `Cycle`
ps: don't forget to update the comment-doc

Reply:
8d4cc94